packages/tools/plymouth/package.mk

```diff
@@ -0,0 +1,100 @@
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+#
+#  OpenELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  OpenELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="plymouth"
+PKG_VERSION="b9f2aac5"
+PKG_REV="1"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="https://cgit.freedesktop.org/plymouth/"
+PKG_URL="https://gitlab.freedesktop.org/plymouth/plymouth/-/archive/master/plymouth-$PKG_VERSION.tar.gz"
+PKG_DEPENDS_INIT="toolchain gcc:init libdrm:init zlib:init libpng"
+PKG_DEPENDS_TARGET="toolchain libdrm zlib libpng"
+PKG_SECTION="tools"
+PKG_SHORTDESC="A graphical boot splash screen with kernel mode-setting support"
+PKG_LONGDESC="A graphical boot splash screen with kernel mode-setting support"
+PKG_TOOLCHAIN="autotools"
+
+PKG_IS_ADDON="no"
+PKG_AUTORECONF="yes"
+
+PLYMOUTH_DRM="--disable-drm"
+if [ "$UVESAFB_SUPPORT" = yes ]; then
+  PKG_DEPENDS_INIT="$PKG_DEPENDS_INIT v86d:init"
```

Comment:
i guess ? @MilhouseVH 
```suggestion
  PKG_DEPENDS_INIT+=" v86d:init"
```

Reply:
Either should be fine, I think `+=` is starting to be used more often now (depending on developer preference), the main thing is to be consistent with existing code (within the same file).